src/common/components/ModifierBar/ModifierBar.tsx

```diff
@@ -15,6 +15,7 @@ export const ModifierBar = () => {
     <div className={cx("displayFlex", "flexGap-medium")}>
       {Object.keys(baseCharacter.modifiers).map((key) => (
         <ModifierItem
+          key={key}
```

Comment:
Any time we have a list we will want to add key. It is not a prop so we don't have to include it on the component just when calling the component.